src/Core/Settings/GlobalSettings.cs

```diff
@@ -545,5 +545,6 @@ public class LaunchDarklySettings : ILaunchDarklySettings
     {
         public string SdkKey { get; set; }
         public string FlagDataFilePath { get; set; } = "flags.json";
+        public Dictionary<string, dynamic> FlagValues { get; set; } = new Dictionary<string, dynamic>();
```

Comment:
I can try it out. I disliked this solution, as well as using `object`. All I want is a non-string-forced way to read these in, parallel to each other.

Reply:
On second thought I imagine these are all `string` under the hood anyways. Since GlobalSettings is bound from `IConfiguration` it all comes from a `Dictionary<string, string>` anyways. Might as well make it `string` for this, it shouldn't make it so that you can do `true` or `10` in the json file though.